code/modules/admin/topic.dm

```diff
@@ -2016,6 +2016,13 @@
 
 		fax_panel(usr)
 
+	else if(href_list["getplaytimewindow"])
+		if(!check_rights(R_ADMIN))	return
+		var/mob/M = locate(href_list["getplaytimewindow"])
+		if(!M)
+			return

```

Comment:
This needs feedback for when a mob can't be found, so admins clicking a non-working link know why. Though, again, it would be preferable if this topic used client references instead of mob references, since clients are what you're really dealing with.

Also, there's a one-line `if() return` above this. Those still exist in the codebase, but shouldn't be used in new code.